### PR TITLE
Quick doc updates for python/console PR

### PR DESCRIPTION
**Pull request by xiazcy**

### CODE REVIEW

File: docs/src/dev/developer/development-environment.asciidoc [hunk 1]
```diff
@@ -39,6 +39,12 @@ of the project and those will go untested.
 To gain the ability to execute all aspects of the TinkerPop build system, other environmental configurations must be
 established. Those prerequisites are defined in the following subsections.
 
+As of TinkerPop 3.5.5, environment configuration for Gremlin Language Variants can be optional, as Docker becomes a system
+requirement to build and test GLVs inside of Maven. Docker enables the building and testing of the GLVs without the need to install GLV-specific environments.
```
Comment (vkagamlyk) [inline docs/src/dev/developer/development-environment.asciidoc:43]: formatting is a bit inconsistent

File: docs/src/dev/developer/development-environment.asciidoc [hunk 2]
```diff
@@ -503,14 +511,16 @@ To be a bit more clear, the version can not be a Docker tag like "latest" becaus
 that has been published with that version number.
 
 [[docker-testing]]
-== Testing with Docker
+== Testing GLVs with Docker
```
Comment (vkagamlyk) [inline docs/src/dev/developer/development-environment.asciidoc:514]: is `gremlin-console` counts and GLV?

File: docs/src/dev/developer/development-environment.asciidoc [hunk 3]
```diff
@@ -39,6 +39,12 @@ of the project and those will go untested.
 To gain the ability to execute all aspects of the TinkerPop build system, other environmental configurations must be
 established. Those prerequisites are defined in the following subsections.
 
+As of TinkerPop 3.5.5, environment configuration for Gremlin Language Variants can be optional, as Docker becomes a system
```
Comment (kenhuuu) [inline docs/src/dev/developer/development-environment.asciidoc:42]: These first two sentences seem to be trying to explain the same thing.

File: docs/src/dev/developer/development-environment.asciidoc [hunk 4]
```diff
@@ -503,14 +511,16 @@ To be a bit more clear, the version can not be a Docker tag like "latest" becaus
 that has been published with that version number.
 
 [[docker-testing]]
-== Testing with Docker
+== Testing GLVs with Docker
 
-Currently gremlin-go, gremlin-javascript, gremlin-dotnet, gremlin-python and gremlin-console can be tested through Docker.
+Currently gremlin-go, gremlin-javascript, gremlin-dotnet, gremlin-python and gremlin-console can be tested through Docker using
```
Comment (kenhuuu) [inline docs/src/dev/developer/development-environment.asciidoc:516]: gremlin-console actually just uses a regular Dockerfile now, and not compose.

File: docs/src/dev/developer/development-environment.asciidoc (hunk 2 above)
Comment (xiazcy) [reply docs/src/dev/developer/development-environment.asciidoc:523]: Not really, but I want to change the title so we don't confuse this with the `Testing with Docker` section in Go...

File: docs/src/dev/developer/development-environment.asciidoc (hunk 1 above)
Comment (xiazcy) [reply docs/src/dev/developer/development-environment.asciidoc]: Will keep consistency with current document

File: docs/src/dev/developer/development-environment.asciidoc (hunk 3 above)
Comment (xiazcy) [reply docs/src/dev/developer/development-environment.asciidoc]: Deleted second one

File: docs/src/dev/developer/development-environment.asciidoc (hunk 4 above)
Comment (xiazcy) [reply docs/src/dev/developer/development-environment.asciidoc]: Updated

File: docs/src/dev/developer/development-environment.asciidoc (hunk 2 above)
Comment (xiazcy) [reply docs/src/dev/developer/development-environment.asciidoc]: Named it `Sub-Modules` instead...

**Pull request merged**